libstemmer/modules.txt

```diff
@@ -29,6 +29,7 @@ italian         UTF_8,ISO_8859_1        italian,it,ita
 lithuanian      UTF_8                   lithuanian,lt,lit
 nepali          UTF_8                   nepali,ne,nep
 norwegian       UTF_8,ISO_8859_1        norwegian,no,nor
+persian         UTF_8                   persian,fa,fas,pers
```

Comment:
I think that should be `per` not `pers` at the end (it should be a three letter code and https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes lists `per` as the ISO 639-2/B code for Persian).